Classes/common/HTTP/CDTURLSession.m

```diff
@@ -53,8 +56,28 @@ - (NSURLSessionDataTask *)dataTaskWithRequest:(NSURLRequest *)request
                                                         NSURLResponse *response,
                                                         NSError *error))completionHandler
 {
+    
+    //make the context
+    CDTURLSessionInterceptorContext *ctx = [[CDTURLSessionInterceptorContext alloc ]initWithRequest:[request mutableCopy]];
+    for ( id obj in self.requestInterceptors) {
+        
+        if( [obj conformsToProtocol:@protocol(CDTURLSessionInterceptor)]){
+            id<CDTURLSessionInterceptor> interceptor = obj;
+            
+            if([interceptor respondsToSelector:@selector(interceptRequestInContext:)]){
+                ctx = [interceptor interceptRequestInContext:ctx];
+            } else {
+                CDTLogWarn(CDTREPLICATION_LOG_CONTEXT, @"%@ doesnn't respond to \
+                           interceptRequestInContext: skipping",interceptor);
```

Comment:
You need some punctuation before `skipping`, I suggest a comma after the method name.